frontend/src/slices/app/actions.ts

```diff
@@ -0,0 +1,18 @@
+import { createAsyncThunk } from '@reduxjs/toolkit';
+
+import { type AsyncThunkConfig } from '~/libs/types/async-thunk-config.type';
+import { type NotificationPayload } from '~/packages/notification/libs/types/notification-payload.type';
```

Comment:
And add root reexport if it's missed
```suggestion
import { type AsyncThunkConfig } from '~/libs/types/types.js';
import { type NotificationPayload } from '~/packages/notification/notification.js';
```